TERMINOLOGY.md

```diff
@@ -1,53 +1,68 @@
-    - Equal Opportunity is a relaxed version of Equalized Odds that only considers positive labels, i.e. Y=1, see [[Hardt et al.]]( https://ttic.uchicago.edu/~nati/Publications/HardtPriceSrebro2016.pdf)
-- Regression:
-    - Statistical Parity: A predictor f satisfies statistical parity under a distribution over (X, A, Y) if f(X) is independent of the group attribute A. Since 0 ≤ f(X) ≤ 1, this is equivalent to P[f(X) ≥ z | A = a] = P[f(X) ≥ z] for all a in A and z in [0,1]. [[Agarwal et al.]]( https://arxiv.org/pdf/1905.12843.pdf)
-    - Bounded Group Loss: A predictor f satisfies bounded group loss at level ζ under a distribution over (X, A, Y ) if E[loss(Y, f(X)) | A = a] ≤ ζ for all a. [[Agarwal et al.]]( https://arxiv.org/pdf/1905.12843.pdf)
+In fairlearn, we provide tools to assess fairness of predictors for classification and regression. We also provide tools that mitigate unfairness in classification and regression. In both assessment and mitigation scenarios, fairness is quantified using disparity metrics as we describe below.
 
-When assessing fairness of models we can measure disparity between groups in a variety of ways. While the criteria above are either fulfilled or not, a disparity metric provides a numerical value with which we can interpret fairness and compare models. We provide the functionality to convert common metrics from scikit-learn to group metrics, i.e. metrics that are evaluated on the entire data set, but also on each group individually. Additionally, group metrics tell us the minimum and maximum metric value and for which groups these values were observed. In certain cases we might be interested in the spread between minimum and maximum metric value, or the ratio between them. All of these are provided by group metric objects. For more information refer to `fairlearn.metrics.metrics_engine.py`.
+#### Group fairness, sensitive features
 
-### Fairness assessment
+There are many approaches to conceptualizing fairness. In fairlearn, we follow the approach known as group fairness, which asks: _Which groups of individuals are at risk for experiencing harms?_
 
-Fairness assessment is the process of determining and evaluating the fairness of a model through a variety of metrics in multiple configurations:
+The relevant groups (also called subpopulations) are defined using **sensitive features** (or sensitive attributes). This term suggests that the system designer should be sensitive to these features when assessing group fairness. Although these features may sometimes have privacy implications (e.g., gender or age) in other cases they may not (e.g., whether or not someone is a native speaker of language). Moreover, the word sensitive does not imply that these features should not be used to make predictions – indeed, in some cases it may be better to include them.
```

Comment:
do we want to explicitly say that we use the name `sensitive_features`?

Reply:
Also, I think it should be "native speaker of _a particular_ language" - except in rare cases, people are generally a "native speaker of language."